modules/end-user-guide/examples/snip_che-supported-languages.adoc

```diff
@@ -1,24 +1,20 @@
-pass:[<!-- vale RedHat.TermsErrors = NO -->]
-
-Apache Camel K:: Stack with tooling ready to develop Integration projects with Apache Camel K
+Apache Camel K:: Stack ready to develop Integration projects with Apache Camel K
 Apache Camel based on Spring Boot:: Stack with environment ready to develop Integration projects with Apache Camel based on Spring Boot.
 Bash:: Stack with environment ready to develop bash scripts.
 C/C++:: Stack with C/C++ and Clang 8
 ASP.NET Core Web Application:: Stack for developing ASP.NET Core Web Application
 Go:: Stack with Go 1.14
 Java Lombok:: Java Stack with Lombok 1.18.18, OpenJDK 11 and Maven 3.6.0
 Java with Spring Boot and MongoDB:: Java stack with OpenJDK 8, MongoDB and Spring Boot Guestbook demo application
-Java Spring Boot:: Java stack with OpenJDK 11 and Spring Boot Petclinic demo application
-NodeJS Angular Web Application:: Stack for developing NodeJS Angular Web Application
-NodeJS Express Web Application:: Stack with NodeJS 10
-NodeJS MongoDB Web Application:: Stack with NodeJS 10 and MongoDB 3.4
-NodeJS React Web Application:: Stack for developing NodeJS React Web Application
-NodeJS Web Application based on Yarn:: Stack for developing NodeJS Web Application based on Yarn 
+Java Spring Boot:: Java stack with OpenJDK 11 and Spring Boot Pet clinic demo application
```

Comment:
In line with https://spring-petclinic.github.io/ ?
```suggestion
Java Spring Boot:: Java stack with OpenJDK 11 and Spring Boot PetClinic demo application
```

Reply:
PetClinic will produce a legit vale error. CamelCase is a programmatic way to write Camel Case. Therefore Pet clinic is legit.